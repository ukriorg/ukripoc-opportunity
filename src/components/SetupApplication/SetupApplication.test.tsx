import React from "react";
import renderer from "react-test-renderer";
import { SetupApplication } from "./index";

// describe("SetupApplication", () => {
//     it("renders correctly", () => {
//         const tree = renderer.create(<SetupApplication />).toJSON();
//         expect(tree).toMatchSnapshot();
//     })
// });
