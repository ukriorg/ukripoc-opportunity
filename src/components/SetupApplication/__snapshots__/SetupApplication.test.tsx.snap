// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SetupApplication renders correctly 1`] = `
<div>
  <div
    className="sc-bwzfXH enMJLA"
  >
    <ol
      className="sc-htpNat kUfvhM"
    >
      <li
        className="sc-bxivhb dQybha"
      >
        <a
          className="sc-bdVaJa haNvij"
          href="/setup/"
        >
          Opportunity setup
        </a>
      </li>
      <li
        className="sc-bxivhb dQybha"
      >
        Website listing
      </li>
    </ol>
  </div>
  <span
    className="sc-ifAKCX gdozZn"
    size="XL"
  >
    
  </span>
  <h1
    className="sc-iwsKbI ftyRPW sc-dnqmqq bFXjfT"
    size="XLARGE"
  >
    Application
  </h1>
  <details
    className="sc-EHOje dqJfCj"
    open={false}
  >
    <summary
      className="sc-bZQynM fdZorV"
    >
      <span
        className="sc-gzVnrw kXCnsC"
      >
        About this workflow component
      </span>
    </summary>
    <div
      className="sc-htoDjs eGnZNt"
    >
      The application component allows you to view and manage applications that are submitted for your opportunity. Using the settings below you can build and customise the application form and also define parameters for application review.
    </div>
  </details>
  <span
    className="sc-ifAKCX kVDkmX"
    size="XL"
  >
    Application settings
  </span>
</div>
`;
